File: node-sdk-examples/completion/applications.js
const Writer = require('writer-sdk');
const dotenv = require('dotenv');

dotenv.config();

const client = new Writer({
    apiKey: process.env.WRITER_API_KEY,
});

async function main() {
    try {
      const response = await client.applications.generateContent(
        process.env.APPLICATION_ID,
        {
        "inputs": [
          {
            "id": "Product description",
            "value": [
              "Terra running shoe"
            ]
          }
        ]
      },
      );
      console.log(response);
    } catch (error) {
      console.error('Error:', error);
    }
  }

main();

File: python-sdk-examples/completion/applications.py
from writerai import Writer
import dotenv
import os
dotenv.load_dotenv()

client = Writer()

def main():
    try:
        application_generate_content_response = client.applications.generate_content(
            application_id=os.getenv("APPLICATION_ID"),
            inputs=[
            {
                "id": "Product description",
                "value": [
                    "Terra running shoe"
                ]
            }
        ],
        )
        print(application_generate_content_response)
    except Exception as error:
        print("Error:", error)

if __name__ == "__main__":
    main()